Rocket.Chat/Extensions/StringExtensions.swift

```diff
@@ -96,7 +96,11 @@ extension String {
     }
 
     func removingWhitespaces() -> String {
-        return components(separatedBy: .whitespaces).joined()
+        return components(separatedBy: .whitespacesAndNewlines).joined()
```

Comment:
Should we update the tests for this class later?

Reply:
✅